app/src/main/res/layout/activity_migration.xml

```diff
@@ -46,16 +47,17 @@
         <androidx.appcompat.widget.AppCompatTextView
                 android:id="@+id/migration_description"
                 android:layout_width="0dp"
+                android:fontFamily="Inter UI"
+                android:justificationMode="inter_word"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="24dp"
                 android:text="@string/migration_description"
                 android:textColor="@color/text_scale_example_text_color"
                 android:textSize="16sp"
-                android:textStyle="bold"
-                android:fontFamily="sans-serif-light"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/migration_firefox_logo" />
+                app:layout_constraintTop_toBottomOf="@+id/migration_firefox_logo"
+                tools:targetApi="o" />
```

Comment:
What does `tools:targetApi="o"` mean here?

Reply:
It's to suppress the warning regarding `android:justificationMode="inter_word"` which needs API level 26 or higher.